index.rst

```diff
@@ -90,30 +90,35 @@ You may also directly use related links to see documents which match you the mos
 
 .. toctree::
    :maxdepth: 1
-   :caption: Application Development
+   :caption: Application Developer's Guide
+
+   Getting Started <appdevguide/gettingStarted>
+   EPICS Overview (Introduction to EPICS) <../getting-started/EPICS_Intro>
+   Build Facility <build-system/specifications>
+   Database Locking, Scanning and Processing (EPICS Process Database Concepts) <process-database/EPICS_Process_Database_Concepts>
+   Database Definition <appdevguide/databaseDefinition>
+   IOC Initialization <appdevguide/IOCInit>
+   appdevguide/AccessSecurity
+   IOC Test Facilities <appdevguide/IOCTestFacilities>
+   Application Developer's Guide (historical, for 3.16.2) <https://epics.anl.gov/base/R3-16/2-docs/AppDevGuide.pdf>
```

Comment:
Can do; I think I mentioned it earlier but removed.

Reply:
OTOH....if it is OK, I will do this in the next step. I already started to do the next set of changes and would prefer to fix this in that step. Otherwise I need to stash my changes, edit, and remember this again when I get back...